lib/exercism/user_statistics.rb

```diff
@@ -0,0 +1,30 @@
+class UserStatistics
+  def self.of(username)
+    by_username(username)
+  rescue Exception => e
+    Bugsnag.notify(e, {username: username})
```

Comment:
If Bugsnag receives a `user` hash, it will stick it in a custom tab. We've done that in [app/routes/core](https://github.com/exercism/exercism.io/blob/master/app/routes/core.rb#L24-L34)

I think I'd rather stick the Bugsnag stuff at the routes level than at the model level. How about adding the same sort of top-level catch in api as we have in app?